astacus/common/utils.py

```diff
@@ -274,11 +276,11 @@ def timedelta_as_short_str(delta):
     return " ".join(f"{v}{u}" for v, u in [(delta.days, "d"), (h, "h"), (m, "m"), (s, "s")] if v)
 
 
-def size_as_short_str(s):
-    for u, m in [("T", 1e12), ("G", 1e9), ("M", 1e6), ("K", 1e3)]:
-        if s >= m:
-            return "%.1f %sB" % (s / m, u)
-    return f"{s} B"
+def size_as_short_str(size: int | float) -> str:
+    for unit, multiplier in [("T", 1e12), ("G", 1e9), ("M", 1e6), ("K", 1e3)]:
+        if size >= multiplier:
+            return f"{size / multiplier:.1f} {unit}B"
```

Comment:
(megacorp policy) No math in f-strings please either.

Reply:
Agree it's slower and less clear to read.